ci-operator/step-registry/rosa/sts/operator-roles/delete/rosa-sts-operator-roles-delete-commands.sh

```diff
@@ -22,10 +22,6 @@ ROSA_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/ocm-token")
 if [[ ! -z "${ROSA_TOKEN}" ]]; then
   echo "Logging into ${OCM_LOGIN_ENV} with offline token"
   rosa login --env "${OCM_LOGIN_ENV}" --token "${ROSA_TOKEN}"
-  if [ $? -ne 0 ]; then
```

Comment:
@yasun1 why are we removing this check everywhere? Any particular reason? Or is the new error message when login fails is more intuitive than this?

Reply:
This is a redundant step, it is never run. If login fails, the script will be terminated at line 24. The failure message from the login is clear enough.

Reply:
Response from Yanming in slack:
This is a redundant step, it is never run. If login fails, the script will be terminated at line 24. The failure message from the login is clear enough.